docs/sources/view-and-analyze-profile-data/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: View and analyze profile data
+menuTitle: View and analyze profile data
+description: How to use Pyroscope to view and analyze profile data.
+aliases:
+  - ../ingest-and-analyze-profile-data/
+weight: 50
+keywords:
+  - pyroscope
+  - UI
+  - performance analysis
+  - flamegraphs
+  - CLI
+---
+
+# View and analyze profile data
+
+Profiling data can be presented in a variety of formats presents such as:
+- **Flamegraphs**: Visualize call relationships and identify hotspots.
+- **Tables**: View detailed statistics for specific functions or time periods.
+- **Charts and graphs**: Analyze trends and compare performance across different metrics.
+
+## Viewing profiles
+
+Pyroscope offers both a Command Line Interface (CLI) and an Application Programming Interface (API) to interact with and retrieve profiling data. These tools provide flexibility in how you access and manage your profiling information.
+
+You can export profiling data from Pyroscope in various formats:
+- **JSON**: JSON object easy to integrate with other tools and scripts
+- **dot**: text containing a DOT representation of the profile
```

Comment:
We used to have this option in pre v1 versions, but as of now there is no way to visualize/export profiles in the dot format in Grafana

Reply:
really? I thought we added it for the hackathon cc @aleks-p

Reply:
There is support in the `/render` endpoint if `format=dot` is passed. Anton is right that we can't visualize it (e.g. with Graphviz) or export it to a file yet.

Reply:
makes sense @knylander-grafana we should remove `dot` from here our options we should list here are `json`, `pprof` 
<img width="333" alt="image" src="https://github.com/grafana/pyroscope/assets/23323466/1a4f391a-1181-446c-b1b6-3a2e6a7fa98f">

(png is actually broken right now)

Reply:
Okay. I'll update it. Thank you! Glad we are doing this review!